renovate.json

```diff
@@ -1,5 +1,10 @@
 {
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
 	"extends": ["github>digicatapult/renovate-config"],
-	"enabled": false
+	"packageRules": [
+		{
+			"matchPackagePatterns": ["^@credo-ts/"],
+			"enabled": false
```

Comment:
I think we want reniovate enabled, just requiring manual approval. Something like this:
```
"packageRules": [
    {
      "matchPackagePatterns": ["^@credo-ts/"],
      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
      "automerge": false
    }
  ]
```